ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -55,19 +56,24 @@ internal class PaywallViewModelImpl(
     override val state: StateFlow<PaywallViewState>
         get() = _state.asStateFlow()
     private val _state: MutableStateFlow<PaywallViewState>
+    private val _lastLocaleList = MutableStateFlow(LocaleListCompat.getDefault())
```

Comment:
`LocaleListCompat.getDefault()` is repeated 3 times in this file, maybe extract it?